### ISSUE TITLE
Using function-like macro with too few arguments segfaults

### ISSUE THREAD

**Issue by nwhetsell** [csound/csound#664]
Running

```csound
#define MACRO(arg1'arg2) #$arg1#
prints "==> $MACRO(argument)\n"
scoreline_i "e"
```

using

```
csound --nosound --orc Test.orc
```

causes a segfault:

```
Csound version 6.07 (double samples) Jun 20 2016
libsndfile-1.0.26
orchname:  Test.orc
defining argument _MACRO__arg1 as...#argument)\n"
scoreline_i "e"

#
defining argument _MACRO__arg2 
csound command: Segmentation fault
end of score.		   overall amps:      0.0
	   overall samples out of range:        0
0 errors in performance
Elapsed time at end of performance: real: 0.251s, CPU: 0.000s
no sound written to disk
```

Using too many arguments does not appear to segfault.

**Comment by jpffitch** [comment]
Kida fixed here; will commit soo

**Comment by nwhetsell** [comment]
For posterity’s sake, this appears to have been addressed in commit ce3ceab.
